README.md

```diff
@@ -36,10 +36,39 @@ Interested in learning more about the basics of virtual environments in Python
 [article](http://www.cs.allegheny.edu/sites/gkapfham/programming/research/idea/2017/07/14/Virtual-Environments/)
 to further develop your understanding of this topic.
 
+GatorGrouper requires users to use Python 3. You can type `python --version` to check
+the current Python version on your device. If you do not have the right Python version,
+or you can not download or upgrade Python on your devices, you can download the
+[Pyenv](https://github.com/pyenv/pyenv) tool to set up a virtual environment for
+the newest Python version.
+
+To install Pyenv, you can use [Pyenv Installer](https://github.com/pyenv/pyenv-installer)
+by typing the command in terminal:
+
+```shell
+curl https://pyenv.run | bash
+```
+
+You will successfully install Pyenv. Please make sure that you have the
+following lines in your `~/.bashrc` file.
```

Comment:
Note that `.bashrc` is one type of file. Others may have a `.zshrc` or other type of file. You can use this as an example. But, I think that somehow, you should generalize this. Finally, while I don't think that we should really address this in the documentation, add this code slows down the startup of a shell &mdash; and so, I wrote my own code to speed up this process, which you can see in my dotfiles repository. There is a lot of content online about how to speed up this kind of code (or, similar code for Ruby or Node). Just a thought, but not something that we have to consider!